src/main/java/duke/exception/DukeInvalidFileException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+import duke.exception.DukeException;
+
+public class DukeInvalidFileException extends DukeException {
+    @Override
+    public String toString() {
+        return "Something wrong with the file";
+    }
+}
```

Comment:
I like how you separate the exceptions into different ones so that it is more clear what type of exception has occured.